src/configimpl.l

```diff
@@ -2064,13 +2065,11 @@ void Config::checkAndCorrect(bool quiet, const bool check)
 
 }
 
-static void appendAttribute(QCString& a, const char * name, ConfigObsolete* value)
+static void appendAttribute(DotAttributes& attr, QCString name, ConfigObsolete* value)
 {
   if (!value || !value->valueStringRef()->length())
     return;
```

Comment:
Probably better to have:
```
if (!value || value->valueStringRef()->isEmpty()) return;
```
so that when accidentally there is just  just white space in the string it will be caught as well.
(I just noticed this now).